lib/segment/src/index/plain_payload_index.rs

```diff
@@ -375,6 +383,11 @@ impl VectorIndex for PlainIndex {
         0
     }
 
+    fn size_of_searchable_vectors_in_bytes(&self) -> Option<usize> {
+        // get the size from the vector storage
+        None
```

Comment:
having implementation returning None makes it very confusing the juggle with the fallback

Reply:
https://github.com/qdrant/qdrant/pull/5587/commits/7733772afafa8195dd85dedd84db843af5857685 :D